packages/dbos-cloud/applications/deploy-app-code.ts

```diff
@@ -42,6 +42,10 @@ export async function deployAppCode(host: string, docker: boolean): Promise<numb
     execSync(`zip -ry ${deployDirectoryName}/${appName}.zip ./* -x ${deployDirectoryName}/* > /dev/null`);
   }
 
+  const interpolatedConfig = readInterpolatedConfig(dbosConfigFilePath)
+  writeFileSync(`${deployDirectoryName}/${dbosConfigFilePath}`, interpolatedConfig)
+  execSync(`zip -j ${deployDirectoryName}/${appName}.zip ${deployDirectoryName}/${dbosConfigFilePath} > /dev/null`);
```

Comment:
Interpolate environment variables into the deployed configuration

Reply:
we can do that *or* we could leverage the real envsubst in the docker container